### PR TITLE
Buff Birdy

### PR DESCRIPTION
**Pull request by tmagicturtle**
As in, improve Birdy. Not like, a super-strong bird or anything.

### CODE REVIEW

File: data/mods/ssb/moves.js [hunk 1]
```diff
@@ -939,7 +939,8 @@ let BattleMovedex = {
 		isNonstandard: "Custom",
 		pp: 5,
 		priority: 0,
-		flags: {dance: 1},
+		flags: {dance: 1, heal: 1},
+		heal: [1, 3],
```
Comment (ghost) [inline data/mods/ssb/moves.js]: causes the move to fail if the user has full HP

Comment (tmagicturtle) [reply data/mods/ssb/moves.js]: Thank you. I've been wondering if there was something wrong with that. I'll fix right away.

File: data/mods/ssb/statuses.js [hunk 2]
```diff
@@ -292,6 +292,19 @@ let BattleStatuses = {
 		onFaint() {
 			this.add(`c|%Birdy~!|My last words will always be that I don't spam.`);
 		},
+		// Feather Tuft Innate
+		onModifyDefPriority: 6,
+		onModifyDef(def, pokemon) {
+			if (!pokemon.transformed && !pokemon.illusion) {
```
Comment (HoeenCoder) [inline data/mods/ssb/statuses.js:298]: Would pokemon.transformed be true when it forme changes?

Comment (ghost) [reply data/mods/ssb/statuses.js:298]: p sure its when the pokemon is transformed

Comment (HoeenCoder) [reply data/mods/ssb/statuses.js:298]: Can this be checked so we can make sure we don't have a sneaky bug on our hand please, maybe add a this.debug call and look for that when testing.

Comment (HoeenCoder) [reply data/mods/ssb/statuses.js:298]: It dosent so were good.

**Pull request merged**